rest-api/model/biobank_order.py

```diff
@@ -125,41 +124,78 @@ class BiobankOrder(BiobankOrderBase, Base):
   samples = relationship('BiobankOrderedSample', cascade='all, delete-orphan')
 
 
-class BiobankOrderIdentifier(Base):
+class BiobankOrderIdentifierBase(object):
+  @declared_attr
```

Comment:
So I believe the only fields you need to do the @declared_attr thing on are foreign keys and relationships. You can leave the other columns the way they were..

Reply:
done